cppwinrt/component_writers.h

```diff
@@ -988,9 +988,22 @@ namespace winrt::@::implementation
     static void write_generated_static_assert(writer& w)
     {
         auto format = R"(
-// Note: Remove this static_assert after copying these generated source files to your project.
-// This assertion exists to avoid compiling these generated source files directly.
-static_assert(false, "Do not compile generated C++/WinRT source files directly");
+// WARNING: This file is automatically generated from IDL. Do not directly
+// add this file to your project, as any changes you make will be lost.
+//
+// If this is a new file:
+//   1. Copy this file from the 'Generated Files' directory into the
```

Comment:
Note that the name "Generated Files" did not come from cppwinrt.exe. It came from the Visual Studio `<GeneratedFilesDir>` property in `Directory.Build.props`. Somebody who uses a different build system (like Razzle) will have an output directory that isn't named "Generated Files".

Perhaps

```suggestion
//   1. Copy this file from its current location into the
```

Reply:
Interesting. The problem is that "current location" is relative, so if you've already moved / copied the file and then "forget" you already did it, the "current location" is actually the proper location. I'll change it to "original location" (which is still not super helpful, but...).